packages/shared/lib/core/profile/utils/hasStrongholdLocked.ts

```diff
@@ -0,0 +1,11 @@
+import { get } from 'svelte/store'
+import { STRONGHOLD_PASSWORD_CLEAR_INTERVAL } from '../constants'
+import { isSoftwareProfile } from '../stores'
+
+export function hasStrongholdLocked(timeSinceStrongholdUnlocked: number): boolean {
+    if (get(isSoftwareProfile)) {
+        return STRONGHOLD_PASSWORD_CLEAR_INTERVAL && timeSinceStrongholdUnlocked > STRONGHOLD_PASSWORD_CLEAR_INTERVAL
```

Comment:
Because if the stronghold status is set to 0 in the backend then it means there is no interval and you can only lock stronghold manually. Should I comment this?

Reply:
ok i thought this constant is never 0. No comment needed